src/include/lexer/lexer.h

```diff
@@ -1,5 +1,5 @@
-#ifndef JUCC_LEXER_H
-#define JUCC_LEXER_H
+#ifndef JUCC_LEXER_LEXER_H
+#define JUCC_LEXER_LEXER_H
```

Comment:
Hey, Good Work :+1:.

@noob77777, do we need to move this Lexer to nested namespace too from `jucc` to `jucc::lexer`?

Reply:
Yes.